app/Jobs/ProcessInboundEmails.php

```diff
@@ -0,0 +1,135 @@
+<?php
+
+namespace App\Jobs;
+
+use App\Models\User;
+use App\Models\Story;
+use App\Enums\StoryStatus;
+use Illuminate\Bus\Queueable;
+use Webklex\IMAP\Facades\Client;
+use App\Mail\CustomerStoryFromMail;
+use Illuminate\Support\Facades\Log;
+use Illuminate\Support\Facades\Mail;
+use App\Mail\OrchestratorUserNotFound;
+use Illuminate\Support\Facades\Storage;
+use Illuminate\Contracts\Queue\ShouldQueue;
+
+class ProcessInboundEmails implements ShouldQueue
```

Comment:
per evitare di intasare orchestrator bisogna prevedere che la mail riceva anche spam o mail al di fuori del sistema di ticketing quindi non tutte le mail devono essere convertite in story prevedere un controllo sulla email mittente che deve coincidere con la mail di chi ha un account su orchestrator

Reply:
creare una funzione apposita che fa il check della email